File: Algorithms/A_Very_Big_Sum/a_very_big_sum.js
// A Very Big Sum


function aVeryBigSum(ar) {
    // Write your code here
    let sum = 0
    for (let i = 0; i < ar.length; i++) {
        const element = ar[i]
        if (typeof element  == "number"){
            sum = sum + element
        }
    }
    console.log(sum)
    return sum
}


const sumThis = [1000000001, 1000000002, 1000000003, 1000000004, 1000000005]

aVeryBigSum(sumThis)

File: Algorithms/A_Very_Big_Sum/a_very_big_sum.py
# A Very Big Sum


def aVeryBigSum(ar):
    # Write your code here
    sum = 0
    for i in ar:
        if type(i) == int:
            sum=sum+i
    print (sum)
    return sum


sumThis = [1000000001, 1000000002, 1000000003, 1000000004, 1000000005]

aVeryBigSum(sumThis)